bokeh/io/notebook.py

```diff
@@ -449,7 +449,7 @@ def load_notebook(resources: Resources | None = None, verbose: bool = False,
         if _NOTEBOOK_LOADED and verbose:
             warnings.append('Warning: BokehJS previously loaded')
 
-        element_id = make_id()
+        element_id: ID|None = make_id()
```

Comment:
I'm wondering where this style of writing `|` is coming from? It's inconsistent with existing usage (even in this module). `black`, for what it's worth, agrees with having spaces around that operator. To me it is much more readable with spaces.

Reply:
```suggestion
        element_id: ID | None = make_id()
```

Reply:
It's not coming from anywhere, it's just neither under automation, nor something I want to devote mental effort to. This is just what happened to come out as I worked in a hurry. FYI if you do want small changes, comment-suggestions are very efficient for everyone.